docs/source/development/sources.rst

```diff
@@ -19,7 +19,7 @@ Ubuntu
 
 .. code-block:: bash
 
-    apt-get install python-pip python-virtualenv python-dev gcc git make realpath screen libffi-dev libssl-dev
+    apt-get install python-pip python-virtualenv python-dev gcc git make realpath screen libffi-dev libssl-dev python3-dev libldap2-dev libsasl2-dev
```

Comment:
I think `python-dev` is not needed anymore as that's replaced with the `python3-dev`

Reply:
Thanks for comments.
Tested on U16 as well, so have removed python-dev, and instead used python3.6-dev so works on both U16 and U18.